drivers/media/i2c/ov9282.c

```diff
@@ -947,6 +956,65 @@ static int ov9282_get_selection(struct v4l2_subdev *sd,
 	return -EINVAL;
 }
 
+/**
+ * ov9282_apply_trigger_config() - Configure sensor for FSIN external trigger mode
+ * @ov9282: pointer to ov9282 device
+ *
+ * Return: 0 on success, error code otherwise.
+ */
+static int ov9282_apply_trigger_config(struct ov9282 *ov9282)
+{
+        int ret;
+
+        ret = ov9282_write_reg(ov9282, OV9282_REG_MODE_SELECT, 1, OV9282_MODE_STANDBY);
+        if (ret) {
+                return ret;
+        }
+
+	/* Low power mode */
+        ret = ov9282_write_reg(ov9282, OV9282_REG_POWER_CTRL, 1, 0x01);
+        if (ret) {
+                return ret;
+        }
+
+	/* External trigger snapshot */
+        ret = ov9282_write_reg(ov9282, OV9282_REG_LOW_POWER_MODE_CTRL, 1, 0x04);
+        if (ret) {
+                return ret;
+        }
+
+	/* 1 frame per trigger */
+        ov9282_write_reg(ov9282, OV9282_REG_NUM_FRAME_ON_TRIG, 1, 0x01);
+        if (ret) {
```

Comment:
You've not saved the result of `ov9282_write_reg` to `ret`, so this will never fail.

Ditto subsequent calls.